ledger/participant-state/kvutils/tools/README.md

```diff
@@ -49,26 +49,28 @@ jmh command line functionality:
 
     bazel run //ledger/participant-state/kvutils/tools:benchmark-replay -- \
       -p ledgerFile=<ledger export files>                                  \
-      -p darFile=<dar files>                                               \
       -p choiceName=<exercise choice names>                                \
-      [-p adapt=true]
+      [-p indexChoice=<index of the choice>                                \
```

Comment:
```suggestion
      [-p choiceIndex=<index of the choice>                                \
```
this has no docs and is probably the one that needs it the most.

I think the name here is also confusing. We take the nth exercise node of that choice, not the nth choice.